iset.mm

```diff
@@ -70653,6 +70661,102 @@ Similar to the Axiom of Choice (first form) of [Enderton] p. 49.
+      NAVBUSULEUBVAULUSUTULPAUSULQUCCUKDUAUDRSAURNUNUPAUQUNBDUTUQNCUKDUTUQPAUSU
+      QQUERSAUPNUPUNAUPPUFAULUQUOTBDKUGUMURUPTGFULUQUOBDUHUIUJ $.
+  $}
+
+  ${
+    $d A u w y z $.  $d B u w y $.  $d ph u w $.
+    exmidordtriimlemb.a $e |- ( ph -> A e. On ) $.
+    exmidordtriimlemb.b $e |- ( ph -> B e. On ) $.
+    exmidordtriimlemb.em $e |- ( ph -> EXMID ) $.
+    exmidordtriimlemb.ha $e |- ( ph -> A. z e. A A. y e. On
+      ( z e. y \/ z = y \/ y e. z ) ) $.
+    exmidordtriimlemb.hb $e |- ( ph -> A. y e. B
+      ( A e. y \/ A = y \/ y e. A ) ) $.
+    $( Lemma for ~ exmidordtriim .  What we get to do based on induction on
+       both ` A ` and ` B ` .  (Contributed by Jim Kingdon, 10-Aug-2024.) $)
+    exmidordtriimlemb $p |- ( ph -> ( A e. B \/ A = B \/ B e. A ) ) $=
+      ( vw vu wcel wceq w3o cv wral con0 wel weq 3mix1 adantl 3mix3 simpr dfss3
+      wa wss sylibr simplr eqssd 3mix2d eleq1 equequ1 3orbi123d ralbidv cbvralv
+      eleq2 sylib eqeq2 rspcv ralimdv sylc biid 3orbi123i 3orcomb 3orrot 3bitri
+      wo eqcom ralbii exmidordtriimlemal adantr mpjaodan ) ADEMZVNDENZEDMZOZKPZ
+      DMZKEQZVNVQAVNVOVPUAUBAVTUFZVPVQLPZEMZLDQZVPVQWAVPVNVOUCUBWAWDUFZVOVNVPWE
+      DEWEWDDEUGWAWDUDLDEUEUHWEVTEDUGAVTWDUIKEDUEUHUJUKAVPWDVHVTALEDFHAWCWBENZE
+      WBMZOZLDQZWGEWBNZWCOZLDQAERMZLBSZLBTZBLSZOZBRQZLDQZWIGACBSZCBTZBCSZOZBRQZ
+      CDQWRIXCWQCLDCLTZXBWPBRXDWSWMWTWNXAWOCPZWBBPZULCLBUMXEWBXFUQUNUOUPURWLWQW
+      HLDWPWHBERXFENWMWCWNWFWOWGXFEWBUQXFEWBUSXFEWBULUNUTVAVBWHWKLDWHWCWJWGOWCW
+      GWJOWKWCWCWFWJWGWGWCVCWBEVIWGVCVDWCWJWGVEWCWGWJVFVGVJURVKVLVMAKDEGHADXFMZ
+      DXFNZXFDMZOZBEQDVRMZDVRNZVSOZKEQJXJXMBKEBKTXGXKXHXLXIVSXFVRDUQXFVRDUSXFVR
+      DULUNUPURVKVM $.
+  $}
+
+  ${
+    $d A b v w y z $.  $d B b $.  $d a b ph u v w $.
+    exmidordtriimlemo.a $e |- ( ph -> A e. On ) $.
+    exmidordtriimlemo.b $e |- ( ph -> B e. On ) $.
+    exmidordtriimlemo.em $e |- ( ph -> EXMID ) $.
+    exmidordtriimlemo.h $e |- ( ph -> A. z e. A A. y e. On
+      ( z e. y \/ z = y \/ y e. z ) ) $.
+    $( Lemma for ~ exmidordtriim .  The induction step for the induction on
+       ` A ` .  (Contributed by Jim Kingdon, 10-Aug-2024.) $)
+    exmidordtriimlemo $p |- ( ph -> ( A e. B \/ A = B \/ B e. A ) ) $=
+      ( vb vw vv cv wcel wceq w3o con0 eqeq2 3orbi123d wral eleq2 wi weq eleq2w
+      eleq1 eleq1w imbi2d wa adantl simpll wem wel simplr simpllr simpr rspcdva
+      mpd ralrimiva cbvralv sylib exmidordtriimlemb exp31 tfis2 impcom ) ADJMZN
+      ZDVEOZVEDNZPZDENZDEOZEDNZPJQEVEEOVFVJVGVKVHVLVEEDUAVEEDRVEEDUESAVIJQVEQNZ
+      AVIAVIUBADKMZNZDVNOZVNDNZPZUBZJKJKUCZVIVRAVTVFVOVGVPVHVQJKDUDVEVNDRJKDUFS
+      UGVMVSKVETZAVIVMWAUHZAUHZBCDVEADQNWBFUIVMWAAUJAUKWBHUIACBULCBUCBCULPBQTCD
+      TWBIUIWCDLMZNZDWDOZWDDNZPZLVETDBMZNZDWIOZWIDNZPZBVETWCWHLVEWCLJULZUHZAWHW
+      BAWNUMWOVSAWHUBKVEWDKLUCZVRWHAWPVOWEVPWFVQWGKLDUDVNWDDRKLDUFSUGVMWAAWNUNW
+      CWNUOUPUQURWHWMLBVELBUCWEWJWFWKWGWLLBDUDWDWIDRLBDUFSUSUTVAVBVCVDURGUP $.
+  $}
+
```

Comment:
I cannot follow the scheme of naming these lemmas: the suffixes are 30, al, b and o - what does this mean? Why not numbering them consecutively starting with *lem1?

Reply:
Short answer is I don't aim for much of a scheme, but when I name them I generally don't know how many there will be or what order they'll go in.

Having said that I'll admit these names are even less descriptive than usual. I guess I'll rename to use consecutive numbers.

Reply:
Done.